### PR TITLE
Fixed behavior when input is an empty array in lido methods

### PR DESCRIPTION
**Pull request by infloop**
Fixed `get_operators_data`, `get_operators_keys`, `validate_keys`, `find_duplicated_keys` methods when passing empty array as input

### CODE REVIEW

**Review by avsetsin — APPROVED** [review]
LGTM

File: lido_sdk/lido.py [hunk 1]
```diff
@@ -61,7 +61,9 @@ def get_operators_data(
         @param operators_indexes: List operators indexes to fetch.
         @return: List of operators details.
         """
-        operators_indexes = operators_indexes or self.operators_indexes
+        operators_indexes = (
+            self.operators_indexes if (operators_indexes is None) else operators_indexes
```
Comment (F4ever) [inline lido_sdk/lido.py:65]:
```suggestion
            self.operators_indexes if operators_indexes is None else operators_indexes
```

File: lido_sdk/lido.py [hunk 2]
```diff
@@ -80,7 +82,7 @@ def get_operators_keys(
         @param operators: List of operators details. We need few fields to fetch "index" and "totalSigningKeys".
         @return: List of keys. Each key can be identified and refetched by "index" and "operator_index".
         """
-        operators = operators or self.operators
+        operators = self.operators if (operators is None) else operators
```
Comment (F4ever) [inline lido_sdk/lido.py:85]:
```suggestion
        operators = self.operators if operators is None else operators
```

File: lido_sdk/lido.py [hunk 3]
```diff
@@ -101,7 +103,7 @@ def validate_keys(
         If not we will try old withdrawal credentials.
         @return: All invalid keys that were found.
         """
-        keys = keys or self.keys
+        keys = self.keys if (keys is None) else keys
```
Comment (F4ever) [inline lido_sdk/lido.py:106]:
```suggestion
        keys = self.keys if keys is None else keys
```

File: lido_sdk/lido.py [hunk 4]
```diff
@@ -118,7 +120,7 @@ def find_duplicated_keys(
         @param keys: List a keys to check.
         @return: List of duplicate pairs keys.
         """
-        keys = keys or self.keys
+        keys = self.keys if (keys is None) else keys
```
Comment (F4ever) [inline lido_sdk/lido.py:123]:
```suggestion
        keys = self.keys if keys is None else keys
```

File: lido_sdk/methods/operators.py [hunk 5]
```diff
@@ -21,6 +21,9 @@ def get_operators_data(w3: Web3, operators_index_list: List[int]) -> List[Operat
     @param operators_index_list: Operator's indexes to fetch
     @return: List of dictionary with operators details
     """
+    if len(operators_index_list) == 0:
```
Comment (F4ever) [inline lido_sdk/methods/operators.py:24]:
```suggestion
    if not operators_index_list:
```

File: lido_sdk/methods/operators.py [hunk 6]
```diff
@@ -41,6 +44,9 @@ def get_operators_keys(w3: Web3, operators: List[Operator]) -> List[OperatorKey]
     """
     args_list = []
 
+    if len(operators) == 0:
```
Comment (F4ever) [inline lido_sdk/methods/operators.py:47]: Hm, this code should be reached. Is it necessary

**Review by F4ever — COMMENTED** [review]
Add changes to CHANGELOG

**Review by F4ever — APPROVED** [review]

**Pull request merged**